logs/models.go

```diff
@@ -13,7 +12,29 @@ var (
 	FatalLevel    = LogLevel{logrus.FatalLevel} // package.
 	ErrorLevel    = LogLevel{logrus.ErrorLevel}
 	TraceLevel    = LogLevel{logrus.TraceLevel}
-	JSONLogFormat = LogFormat{&logmatic.JSONFormatter{}}
+	JSONLogFormat = LogFormat{&logrus.JSONFormatter{
+			DisableTimestamp:			false,
+			TimestampFormat:			"2006-01-02 15:04:05",
+			DataKey:					"",
+			FieldMap:					nil,
+			CallerPrettyfier:			nil,
+			PrettyPrint:				false,
+	}}
+	TextLogFormatter = LogFormat{&logrus.TextFormatter{
```

Comment:
Let's create an issue to create a config to edit all these values via configuration file or flags.

Reply:
Make sure you use `gofmt` before pushing to format to spec

Reply:
@andrewnguyen22  
https://github.com/pokt-network/pocket-core/pull/441/commits/349a75aa0c292f76903432011bd873af328cfa51

@luyzdeleon 
https://github.com/pokt-network/pocket-core/pull/441/commits/c99e86fbb9d9403ec280be779c2d090a9ca688e5